### ISSUE TITLE
Accessibility: showInputBox title is not read aloud by screen reader

### ISSUE THREAD

**Issue by BenBeckley** [microsoft/vscode#242581]
Does this issue occur when all extensions are disabled?: No

- VS Code Version: 1.97.2
- OS Version: Windows 11 Version 24H2

Steps to Reproduce:

1. Enable a screen reader such as NVDA or JAWS. 
2. Enable Screen Reader Accessibility mode in VS Code.
3. Start VS Code and invoke a command that has an Input Box with a title*

Expected: The title of the input box should be read aloud by the screen reader
Actual: The title of the input box is not read aloud
Impact: Users utilizing screen readers may experience difficulty in identifying the names of edit fields clearly.
*An example of this is the [VS Code Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions), and the "Create Function App in Azure..." command. 

<img width="611" alt="Image" src="https://github.com/user-attachments/assets/90ce728d-b134-4a3d-ae3d-11b3804314a9" />

**Comment by TylerLeonhardt** [comment]
Yeah at some point this was read out... but for some reason it's not now.

**Comment by TylerLeonhardt** [comment]
Fixed in https://github.com/microsoft/vscode/pull/246497
